src/main/java/net/fabricmc/loom/api/mappings/layered/spec/FileSpec.java

```diff
@@ -42,17 +44,35 @@
  */
 @ApiStatus.Experimental
 public interface FileSpec {
+	/**
+	 * Creates a file spec.
+	 *
+	 * <p>The parameter will be evaluated like this:
+	 * <ul>
+	 * <li>{@link File}, {@link Path} and {@link org.gradle.api.file.FileSystemLocation} will be resolved as local files</li>
```

Comment:
```suggestion
	 * <li>{@link File}, {@link Path} and {@link FileSystemLocation} will be resolved as local files</li>
```
It's already imported, so FQN not necessary